client/app/pages/queries/query.html

```diff
@@ -217,7 +217,9 @@ <h3>
               <!-- End of Query Execution Status -->
 
               <!-- tabs and data -->
-              <div ng-if="showDataset" class="flex-fill p-relative">
+              <div ng-if="showDataset && dataSourceChanged" class="flex-fill p-relative">
+                <div class="d-flex flex-column p-absolute" style="left: 0; top: 0; right: 0; bottom: 0;">
```

Comment:
This opens a new `div` without closing it, and also uses inline styles. Is this needed or just for debugging purposes?

Reply:
These lines actually look like the result of a bad merge as they serve no purpose. I've removed them. The in-line styles below were there before me.